.ansible/roles/cosmos-faucet/defaults/main.yml

```diff
@@ -0,0 +1,8 @@
+faucet_version: master
+log_level: info
+key_name: faucet
+cli_name: gaiacli
```

Comment:
What gaia is used for here?

Reply:
great catch! thank you! It should be `spnd`

Reply:
oh, actually those are the default values, as that role was initially designed for `gaia`. The value is overwritten [here](https://github.com/tendermint/spn/blob/helder/add-deploy-action/.ansible/playbook.yml#L58)

Reply:
Ok great 👍